manifests/pipecd/templates/configmap.yaml

```diff
@@ -200,6 +200,8 @@ data:
             replacement: $1
 
       - job_name: pipecd-ops
+        # This scrape_interval must not be larger than 1m, which is the interval of piped's stats-reporting.
```

Comment:
Should mention it as not less than piped stat-reporting interval is enough. We may change the interval of that piped stat reporting later

Reply:
@khanhtc1202 
Thanks, I agree with you.
I fixed as below:

>    # This scrape_interval must not be larger than the interval of piped's stats-reporting.